templates/promo.html

```diff
@@ -0,0 +1,112 @@
+<div>
```

Comment:
I tried that, JSX is mangling the <defs> inside the <svg>, resulting in gradients etc. not rendering correctly.